src/test/java/integ/com/mathworks/ci/BuildArtifactActionTest.java

```diff
@@ -20,6 +20,8 @@
 import org.junit.Test;
 import org.jvnet.hudson.test.JenkinsRule;
 
+import com.mathworks.ci.freestyle.RunMatlabBuildBuilder;
+
 public class BuildArtifactActionTest {
```

Comment:
Yeah I'd be happy to do that in the future! For now though, because `BuildArtifactAction`'s code hasn't been touched by these changes and none of the tests covering it have been removed I think I'm going to leave that out of scope for this particular PR since it's already on the larger side. 

I want to revisit and update some of the integ tests in the future as well, so I can make that change when I get back to do those tests.